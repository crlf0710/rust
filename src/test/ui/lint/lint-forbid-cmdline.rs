// check-pass
// compile-flags: -F deprecated

#[allow(deprecated)] //~ WARNING allow(deprecated) overruled by outer forbid(deprecated)
fn main() {
}
