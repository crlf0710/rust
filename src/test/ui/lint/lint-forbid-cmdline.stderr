warning[E0453]: allow(deprecated) overruled by outer forbid(deprecated)
  --> $DIR/lint-forbid-cmdline.rs:4:9
   |
LL | #[allow(deprecated)]
   |         ^^^^^^^^^^ overruled by previous forbid
   |
   = note: `forbid` lint level was set on command line

warning: 1 warning emitted

For more information about this error, try `rustc --explain E0453`.
